bspump/elasticsearch/connection.py

```diff
@@ -47,18 +47,15 @@ def __init__(self, connection, index, max_size):
 		self.FailLogMaxSize = connection.FailLogMaxSize
 		self.FilterPath = connection.FilterPath
 
-		# Get credentials
-		username = connection.Config.get('username')
-		password = connection.Config.get('password')
-		api_key = connection.Config.get('api_key')
-		url = connection.Config.get('url')
+		# Get credentials for authorization and nodes' urls
+		username, password, api_key, node_urls = get_credentials_from_config(connection.Config)
```

Comment:
I don't understand why this "Bulk" object is reading the configuration in the first place